migrator.php

```diff
@@ -1240,7 +1256,29 @@ private function create_or_update_woo_order( $shopify_order, $woo_order ) {
 		$this->process_order_refunds( $order, $shopify_order );
 	}
 
-	private function create_or_assign_customer( $order, $shopify_order ) {
+	/**
+	 * Scramble email address
+	 * Used for test mode order import
+	 *
+	 * @param string $email
+	 * @return string
+	 */
+	private function scrambleEmail( $email ) {
```

Comment:
We should use snake case for functions and variables

Reply:
Thanks for the call-out. I have removed this function for now, since we are only adding a `.masked`, as per our discussion on slack.